trino-config/catalog/iceberg.properties

```diff
@@ -0,0 +1,6 @@
+connector.name=iceberg
+iceberg.catalog.type=rest
+iceberg.rest-catalog.uri=http://localhost:8181/api/catalog
+iceberg.rest-catalog.security=OAUTH2
+iceberg.rest-catalog.oauth2.token=principal:root;password:e6d4d1a345e900dc9c82d64d64e075ed;realm:default-realm;role:ALL
```

Comment:
Feels weird that we would require this in the config, not sure what other solutions there are here but I would hope this would work out of the box with our other docker image. Should this maybe be a required setup script? or Users should be forced to set this?

Reply:
I agree; I wanted to call this out as a friction point in the current setup.
AFAIK, the user is required to set this since Polaris vend this credential at start up